packages/components/ContextualMenu/src/ContextualMenu.types.ts

```diff
@@ -0,0 +1,53 @@
+import { IRenderData } from '@uifabricshared/foundation-composable';
+import { ICalloutProps, ICalloutTokens } from '@fluentui-react-native/callout';
+
+export const contextualMenuName = 'ContexualMenu';
+
+/*
+export interface IContextualMenuContext {
+  /*
```

Comment:
I know this is an early version, but do we want to have large chunks of commented out code left here? Maybe remove it from the PR but keep it locally?